news/_posts/2020-call-for-paper-quick-tour.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "2020 Call for Papers: that was great!"
+subtitle: "A quick tour of the result"
+date:  2020-02-21 16:30:00
```

Comment:
```suggestion
date:  2020-11-27 14:00:00
```

Reply:
Times are UTC

Reply:
I was waiting for the review to update the date